### PR TITLE
Bitrise step eschecker 0.0.1

### PR DESCRIPTION
**Pull request by Treyone**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

**Comment by Treyone** [comment]
@ofalvai I used the CLI for step creation but totally missed the `stepman` tool in the first place and did everything manually. I fixed that and followed the guidelines but again, validation fails and I don't know where to find information on what causes the error...

File: steps/bitrise-step-eschecker/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    is_required: true
+    summary: Unique ID of the campaign you can to run in esChecker
+    title: Campaign ID
+outputs:
+- eschecker_pdf_report: null
+  opts:
+    description: |
+      PDF export of the esChecker report.
+    summary: Contains all relevant information oo an esChecker Campaign.
+    title: esChecker report PDF
```
Comment (koral--) [inline steps/bitrise-step-eschecker/0.0.1/step.yml:55]: Is it a path in local file system or an URL?
It should be described explicitly.

**Comment by Treyone** [comment]
Thanks, I updated that and bumped to 0.0.2 in https://github.com/bitrise-io/bitrise-steplib/pull/3690.

File: steps/bitrise-step-eschecker/0.0.1/step.yml (hunk 1 above)
Comment (Treyone) [reply steps/bitrise-step-eschecker/0.0.1/step.yml:55]: Done in new PR, thanks for the hint!

**Pull request closed without merging**